Gruntfile.js

```diff
@@ -203,12 +203,14 @@ module.exports = function( grunt ) {
 
 	grunt.registerTask( "lint", [
 		"jsonlint",
-		runIfNewNode( "eslint" )
+		runIfNewNode( "eslint:dev" ),
+		runIfNewNode( "eslint:dist" )
 	] );
 
 	grunt.registerTask( "lint:newer", [
 		"newer:jsonlint",
-		runIfNewNode( "newer:eslint" )
```

Comment:
Same idea as in https://github.com/jquery/jquery-migrate/pull/238/files#diff-35b4a816e0441e6a375cd925af50752cR129 I suppose? 

I wouldn't be oppose to comment about this in the code